examples/Web/RequestTimerMiddleware.cs

```diff
@@ -0,0 +1,49 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Configuration;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Confluent.Kafka;
+
+
+namespace Web
+{
+    /// <summary>
+    ///     Middleware that times how long a web request takes to be handled,
+    ///     and logs this to Kafka.
+    /// </summary>
+    public class RequestTimerMiddleware
+    {
+        private readonly string topic;
+        private readonly KafkaDependentProducer<string, long> producer;
+        private readonly RequestDelegate next;
+
+        public RequestTimerMiddleware(RequestDelegate next, KafkaDependentProducer<string, long> producer, IConfiguration config)
```

Comment:
docstring?   next is so-n-so

Reply:
`next` is what is conventionally used.

doc string on the class is enough i think. this is a standard pattern.